benchopt/runner.py

```diff
@@ -290,9 +291,23 @@ def run_one_solver(benchmark, objective, solver, meta, max_runs, n_repetitions,
                 progress_str=progress_str, solver=solver
             )
 
-            solver_strategy = getattr(
-                solver, 'stop_strategy', solver.stopping_criterion.strategy
-            )
+            if hasattr(solver, 'stop_strategy'):
+                warnings.warn(
+                    "'stop_strategy' attribute is deprecated \
+                     use 'stopping_strategy' instead",
+                    DeprecationWarning
+                )
+                solver_strategy = getattr(
+                    solver,
+                    'stop_strategy',
+                    solver.stopping_criterion.strategy
+                )
+            else:
+                solver_strategy = getattr(
+                    solver,
+                    'stopping_strategy',
+                    solver.stopping_criterion.strategy
+                )
```

Comment:
Maybe use a property for `BaseSolver` with this code, to avoid duplicating this in `stopping_criterion.py`?

Reply:
I added a property named "stopping_strategy_api" in BaseSolver